ceph-releases/kraken/redhat/7.2/Dockerfile

```diff
@@ -0,0 +1,49 @@
+FROM  registry.access.redhat.com/rhel7.2
```

Comment:
This one is purely Red Hat let's remove it as some of the links won't work, I'll add the support for Red Hat :).